sunxspex/emission.py

```diff
@@ -239,6 +274,62 @@ def bremsstrahlung_cross_section(electron_energy, photon_energy, z=1.2):
     return cross_section
 
 
+def brm2_fthin(electron_energy, photon_energy, eelow, eebrk, eehigh, p, q, z=1.2, efd=True):
+    """
+    Returns the integrand for the thin-target bremsstrahlung integration.
+
+    Parameters
+    ----------
+    electron_energy : np.array
+        Electron energies in keV. Two-dimensional array of abscissas calculated in brm_gauss_legendre()
+    photon_energy : np.array
+        Photon energies in keV
+    eelow : float
+        Low energy electron cut off
+    eebrk : float
+        Break energy
+    eehigh :
+        High energy cutoff
+    p : float
+        Slope below the break energy
+    q : float
+        Slope above the break energy
+    z : float
+        Mean atomic number of plasma
+    efd: bool
+        If set to True (default), the electron flux distribution is calculated from broken_powerlaw().
+        If set to False, the electron density distribution is calculated from broken_powerlaw().
+
+    Returns
+    -------
+    np.array
+        Bremsstrahlung photon flux at given photon energies
+
+    Notes
+    -----
+
+    History
+    ------
+    Mar 13, 2020, binchen@njit.edu, initial version modified from SSW Brm2_FThin.pro:
+        https://hesperia.gsfc.nasa.gov/ssw/packages/xray/idl/brm2/brm2_fthin.pro
+    """
+    mc2 = 510.98
```

Comment:
Have a lootk at the way the constants are done in the thicktarget function below

Reply:
done